packages/connect-voting-disputable/src/models/Vote.ts

```diff
@@ -67,8 +85,45 @@ export default class Vote {
   }
 
   votingPowerPct(num: string): string {
-    const votingPower = BigNumber.from(this.votingPower)
-    return BigNumber.from(num).mul(BigNumber.from(100)).div(votingPower).toString()
+    const votingPower = bn(this.votingPower)
+    return bn(num).mul(PCT_BASE).div(votingPower).toString()
+  }
+
+  async formattedYeas(): Promise<string> {
+    return formatBn(this.yeas, await this._tokenDecimals())
```

Comment:
What you think of having the `tokenDecimals` be part of the Vote entity at the subgraph level. I wonder if we may prevent a lot of request to fetch the decimals value that's something we are sure is not going to change after the creation of the token.

Reply:
I like the idea :)